resources/js/mp-history/game-header.tsx

```diff
@@ -24,7 +24,7 @@ const timeFormat = 'LTS';
 export default function GameHeader(props: Props) {
   let title = getTitle(props.beatmapset);
   const version = props.beatmap.version;
-  if (version != null) {
+  if (version !== '') {
```

Comment:
This is forced by changes to `deletedBeatmap()` in `Game` - previous code would put a version of `null` when using the fallback to a deleted beatmap, typings be damned, while now I've had to make the function obey to the typings, and `null` is no longer permitted in the `version` field, so empty string is the next best thing.